examples/layouting.jl

```diff
@@ -486,4 +486,95 @@ end
 
     end
 
+    @cell "markers, an arraow, and ellipses in CairoMakie" [layout] begin
+
+        using DataFrames, Distributions, Colors
```

Comment:
If these packages aren't in MakieGallery's dependencies already, we should add them

Reply:
They are already in MakieGallery's dependencies, so all good. I checked.